app/android/app/src/main/kotlin/com/example/app/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.app
```

Comment:
Is this file the "dummy file" for launching an empty app? Looks like it, good to note for future reference

Reply:
oh yea, this package. When you create flutter project, Android studio asks a package name. I'm not sure what to name for our project package name, so I leave it as example. I'll remove/change this in the future commits